lib/hackmode/config.js

```diff
@@ -1,6 +1,18 @@
 const { config } = require('../n3h-common')
+const mosodium = require('../mosodium')
 
 module.exports = exports = config.createDefinition({
+  network: {
+    id: config.entry({
+      type: 'string',
+      default: 'holochain closed alpha testnet d5d6c38c-648e-4ffc-b098-75d4181e639b',
```

Comment:
It's not hardcoded, it's set as the default : )

This is what will allow us to create intentionally partitioned test-nets. It's in the in-transit encryption roadmap. Once we get the node-2-node encryption in place, two nodes with different network id hashes will refuse to communicate with each other.

It also makes sense to base the mdns discovery service name off of this, so that two nodes on different networks will not auto-discover each other. Which is why it's added in this PR.

Reply:
Cool thanks. Yeah, I just meant hardcoded value as in the default.